src/main/scala/mesosphere/marathon/upgrade/GroupVersioningUtil.scala

```diff
@@ -28,17 +28,17 @@ object GroupVersioningUtil {
     def updateAppVersionInfo(maybeOldApp: Option[AppDefinition], newApp: AppDefinition): AppDefinition = {
       val newVersionInfo = maybeOldApp match {
         case None =>
-          log.info(s"[${newApp.id}]: new app detected")
+          log.info(s"${newApp.id}: new app detected")
```

Comment:
I would keep logging statements stable as long as there is no additional value in changing them. Probably we do have users/customers running regex on those logs and maybe they need `[` and `]`. But I understand the wish to align the messages with others...I don`t have strong feelings about one or the other solution, maybe the other reviewers? :)

Reply:
This is the only place we use brackets `[]`. Anyway, "keeping logs stable" is not part of our contract with customers - otherwise this argument can be applicable to any other change in this PR ;)

Reply:
Any other opinions? 📦 

// I`m ok with a merge 😘